### PR TITLE
Update swagger.yaml to latest version

### LINKED ISSUE #656: Do we need to update swagger.yaml to latest version?
**What is the problem?**
`swagger.yaml` only supports 2.13 version, and it's out of date compared to current specification.

**Who does this affect?**
Currently I'm working for [https://github.com/openservicebrokerapi/osb-checker/pull/77](https://github.com/openservicebrokerapi/osb-checker/pull/77), and I want to leverage `go-swagger` tool to generate source code automatically from specification, but then I found the `swagger.yaml` only supports v2.13.

**Do you have any proposed solutions?**
If there is no concern to modify `swagger.yaml` file, I'm willing to help update it to latest version.

**Additional context**
Add reviewers: @mattmcneeney @fmui @jberkhahn @tinygrasshopper

### PR DESCRIPTION
**Pull request by leonwanghui**
## Introduction
This PR is proposed to fix #656 .
Please take a look at [swagger UI](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/leonwanghui/servicebroker/swagger-update/swagger.yaml) to check it.

Add some reviewers: @mattmcneeney @fmui @jberkhahn @tinygrasshopper

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -17,23 +17,39 @@ info:
     name: Open Service Broker API
     url: 'https://www.openservicebrokerapi.org/'
     email: open-service-broker-api@googlegroups.com
-  version: v2.13
+  version: master - might contain changes that are not yet released
+
+host: localhost:3000
```
Comment (mattmcneeney) [inline swagger.yaml]: Should we have a `host` in here?

File: swagger.yaml [hunk 2]
```diff
@@ -17,23 +17,39 @@ info:
     name: Open Service Broker API
     url: 'https://www.openservicebrokerapi.org/'
     email: open-service-broker-api@googlegroups.com
-  version: v2.13
+  version: master - might contain changes that are not yet released
```
Comment (mattmcneeney) [inline swagger.yaml]: The tabbing looks a little off here

Comment (leonwanghui) [reply swagger.yaml]: I just copied it from `openapi.yaml`, do you have any suggestion?

Comment (mattmcneeney) [reply swagger.yaml]: It looks fine in this view now - I think it's OK!

File: swagger.yaml (hunk 1 above)
Comment (leonwanghui) [reply swagger.yaml]: I saw `servers` is contained in `openapi.yaml`, so I added this `host` field to keep up with it.

**Comment by leonwanghui** [comment]
@mattmcneeney @fmui @jberkhahn @tinygrasshopper Please take a review at this PR, thanks!

**Pull request merged**